### PR TITLE
Remove use of nn.stochastic from seq2seq example

### PR DESCRIPTION
**Pull request by jheek**

### CODE REVIEW

File: examples/seq2seq/train.py [hunk 1]
```diff
@@ -169,12 +168,12 @@ def apply(self,
...
 
 
-def create_model():
+def create_model(rng):
   """Creates a seq2seq model."""
   vocab_size = CTABLE.vocab_size()
   _, initial_params = Seq2seq.init_by_shape(
-      nn.make_rng(), [((1, get_max_input_len(), vocab_size), jnp.float32),
-                      ((1, get_max_output_len(), vocab_size), jnp.float32)])
+      rng, [((1, get_max_input_len(), vocab_size), jnp.float32),
```
Comment (marcvanzee) [inline examples/seq2seq/train.py]: Any reason why you don't just write "jax.random.PRNGKey(0)" here directly, and avoid passing it as an argument to create_model?

Comment (jheek) [reply examples/seq2seq/train.py]: I find it convenient to keep the global random seed at the top level so that it can be split up from there if necessary or perhaps be initialized using a flag like `rng = random.PRNGKey(FLAGS.seed)`.

Comment (marcvanzee) [reply examples/seq2seq/train.py]: Ok, makes sense.
